README.md

```diff
@@ -30,6 +30,7 @@ To build this plugin you need to set up following environments variable:
 - `CLIENT_REGISTER_URI_PATH_NAME` - Optional - Client URI path name as entry point to generate PP register URI and redirect to it, ie: "/register-overlay/" 
 - `CLIENT_LOGIN_URI_PATH_NAME` - Optional - Client URI path name as entry point to generate PP login URI and redirect to it, ie: "/login-overlay/"  
 - `CLIENT_FORGOT_PASSWORD_URI_PATH_NAME` - Optional - Client URI path name as entry point to generate PP forgot password URI and redirect to it, ie: "/forgot-password-overlay/"
+- `FORCE_REDIRECT` - Optional - force redirect event if user already logged in, set 'true' for force redirect"    
```

Comment:
How if we change the description to be more explicit like this,
`force redirect to player portal on login button click if the player already logged in`

Reply:
I think `on login button click` is not right statement since when user already login the button show username
this redirect is related to https://github.com/AccelByte/wordpress-react-plugin/pull/3 feature
wdyt @THZs?

Reply:
Oh sorry, I thought it was in the login button.
I think we need to change the variable name to be like this `CLIENT_OVERLAY_URI_PATH_AUTO_REDIRECT` to make it more explicit and easy to understand.

- `CLIENT_OVERLAY_URI_PATH_AUTO_REDIRECT` - Optional - force redirect to player portal page if the user already logged in when accessing `CLIENT_REGISTER_URI_PATH_NAME`, `CLIENT_FORGOT_PASSWORD_URI_PATH_NAME` and `CLIENT_LOGIN_URI_PATH_NAME`, (default `true`)

Reply:
var name updated